src/main/java/com/sergiofah/integracao/controller/ProductPageController.java

```diff
@@ -45,53 +52,70 @@ public class ProductPageController {
 	
 	@FXML
 	private ImageView productImageView;
+	private List<Line> lineList;
+	private List<Category> categoryList;
+	private List<Product> productList = new ArrayList<>();
 
-    public ProductPageController() {
+	public ProductPageController() {
         loading = new Image(getClass().getResourceAsStream("/images/loading.gif"));
     }
 
     @FXML
 	private void initialize() {
-		productDAO = new ProductDAO();
-		populateComboBox();
+		this.service = new Service();
+		if(service.getServerStatus()) {
+			this.lineList = service.getLines();
+			populateComboBox();
+		}else{
+			serverStatus.setText("Status: Server Offline");
+			serverStatus.setTextFill(Color.RED);
+		}
 	}
 
 	public void populateComboBox(){
```

Comment:
The left brace must always be separated from the previous character by most Java code conventions and our convention.